djangoProject/apiDrfDocker/apps.py

```diff
@@ -0,0 +1,6 @@
+from django.apps import AppConfig
+
+
+class ApidrfdockerConfig(AppConfig):
```

Comment:
```suggestion
class ApiDRFDockerConfig(AppConfig):
```
According to the PEP8 guidelines

Reply:
@tucosaurus  ok. i didn't read the document. i was thinking of reading it tomorrow . i'll do that asap and make the necessary changes here.